spec/command_recorder_spec.rb

```diff
@@ -37,7 +37,7 @@ def before(env)
     end
 
     it "does not fail when reverting" do
-      migration = Class.new ::ActiveRecord::Migration do
+      migration = Class.new ::ActiveRecord::Migration[5.0] do
```

Comment:
> Strange that enabled_middleware always returns false. Are you sure? (Actually it should return either a class or nil. See spec/support/enableable.rb

Sorry, I had meant falsey (i.e. `nil`), not `false`.

> (in this case, the call to enabled_middleware always returns false)

This statement was actually incorrect. I took a fresh look at it this morning. All appending `[5.0]` to `::ActiveRecord::Migration` does is fix a deprecation warning.